subprojects/snapshots/src/integTest/groovy/org/gradle/internal/vfs/VirtualFileSystemRetentionIntegrationTest.groovy

```diff
@@ -48,21 +47,15 @@ class VirtualFileSystemRetentionIntegrationTest extends AbstractIntegrationSpec
 
         when:
         mainSourceFile.text = sourceFileWithGreeting("Hello VFS!")
+        Thread.sleep(2100)
```

Comment:
Could you extract that constant/method so it is obvious why we sleep 2.1 seconds in various places?

Reply:
Good point.